Location/Location-Nightingale-RR805.json

```diff
@@ -42,7 +42,7 @@
             ],
             "coding": [
                 {
-                    "system": "https://fhir.nhs.uk/CodeSystem/organisation-role",
+                    "system": "https://fhir.nhs.uk/CodeSystem/England-organisation-role",
```

Comment:
Do we have an agreed naming convension for extensions, CodeSystems and ValueSets? 
On a previous call with Simplifier, they noted that resources with the same ID will have 'duplicate' appended in Simplifier, so it may be best to call out the resource type within the URL/ID. I would also advise the last part of the URL indicating the name is not given hypens and is specified in PascalCase, to mirror the resource naming convension where the resource type spans multiple words. So for the CodeSystem here I would change this to "CodeSystem-England-OrganizationRole"

Reply:
There is a naming convention in the Design and Development Approach but not yet watertight and also doesn't state your suggestion of CodeSystem-England-[Name] however I do like your suggestion. I'll speak with KH et al today to see if we can agree a way forward with it at least temporarily.